woocommerce-gateway-stripe.php

```diff
@@ -250,10 +260,16 @@ public function init() {
 				require_once dirname( __FILE__ ) . '/includes/class-wc-stripe-account.php';
 				new Allowed_Payment_Request_Button_Types_Update();
 
-				$this->api                           = new WC_Stripe_Connect_API();
-				$this->connect                       = new WC_Stripe_Connect( $this->api );
-				$this->payment_request_configuration = new WC_Stripe_Payment_Request();
-				$this->account                       = new WC_Stripe_Account( $this->connect, 'WC_Stripe_API' );
+				$this->api                            = new WC_Stripe_Connect_API();
+				$this->connect                        = new WC_Stripe_Connect( $this->api );
+				$this->payment_request_configuration  = new WC_Stripe_Payment_Request();
+				$this->account                        = new WC_Stripe_Account( $this->connect, 'WC_Stripe_API' );
+
+				// Express checkout configurations.
+				$express_checkout_helper = new WC_Stripe_Express_Checkout_Helper();
+				$express_checkout_ajax_handler = new WC_Stripe_Express_Checkout_Ajax_Handler( $express_checkout_helper );
+				$this->express_checkout_configuration = new WC_Stripe_Express_Checkout_Element( $express_checkout_ajax_handler, $express_checkout_helper );
```

Comment:
There are some minor alignment issues here and the block above. They all might be aligned with the `$this->express_checkout_configuration = ...`

But the whole block should be together or just aligned with their block.

Reply:
Fixed it in the other PR [4dcfe4a](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3439/commits/4dcfe4a39b36e53b4dca89c99304f43285bcf1e0)